lib/themes/dosomething/paraneue_dosomething/scss/patterns/_button.scss

```diff
@@ -1,14 +1,73 @@
+%animated {
```

Comment:
I can't cause I'm changing the animation duration. I probs don't need the placeholder anymore since it's only one rule now... before I was using both the fill-mode and the duration...